src/content/advanced/gitops/index.md

```diff
@@ -125,367 +123,195 @@ flux-multi-tenancy:
-NAME                        READY   STATUS                             AGE
-install-cluster-aws-chart   True    Release reconciliation succeeded   9s
-
-kubectl get clusters.cluster.x-k8s.io -n org-flux-demo demo0
-NAME    PHASE
-demo0
+kubectl create -f management-clusters/MC_NAME/MC_NAME.yaml
 ```
 
-### Installing managed apps
-
-It is just as easy to install managed apps in existing workload clusters. In this part of the guide we will assume you have completed the previous steps and have both an organization and a cluster running.
-
-File structure
+### Managing organizations
 
-```nohighlight
-.
-├── 04-managed-app.yaml     # contains Kustomization CR
-├── 04-managed-app
-│   ├── kustomization.yaml
-│   └── resources.yaml      # contains Giant Swarm App CR
-```
+The highest level Giant Swarm custom resource you can create is an [Organization]({{< relref "/ui-api/management-api/crd/organizations.security.giantswarm.io.md" >}}).
 
-```nohighlight
-mkdir 04-managed-app
-```
+In order to create the files in the repository, run the command (in the root of the repo):
 
 ```nohighlight
-echo -e 'resources:\n- resources.yaml' > 04-managed-app/kustomization.yaml
+kubectl gs gitops add organization --name ORG_NAME --management-cluster MC_NAME
 ```
 
-```nohighlight
-kubectl gs template app \
-        --cluster "demo0" \
-        --catalog giantswarm \
-        --name nginx-ingress-controller-app \
-        --namespace default \
-        --version 2.4.0 > 04-managed-app/resources.yaml
-```
+This is enough to create the Organization on our own. It creates automatically the organization namespace where your cluster resources will be applied.
 
 ```nohighlight
-flux create kustomization install-managed-app \
-        --source=GitRepository/flux-demo.default \
-        --service-account=automation \
-        --path="./04-managed-app" \
-        --prune=true  \
-        --interval=30s \
-        --export > 04-managed-app.yaml
+kubectl get namespaces (names will difer)
+NAME            STATUS   AGE
+org-acme   Active   2m29s
 ```
 
-Install the `Kustomization`:
-
-```nohighlight
-kubectl create -f 04-managed-app.yaml
-kustomization.kustomize.toolkit.fluxcd.io/install-managed-app created
-
-kubectl get kustomizations -n default install-organization
-NAME                  READY   STATUS                                                            AGE
-install-managed-app   True    Applied revision: main/070c352f4949ecaad0d43e31744adf7e864cc754   23s
-
-kubectl get apps -n demo0 nginx-ingress-controller-app
-NAME                                VERSION      LAST DEPLOYED   STATUS
-nginx-ingress-controller-app        2.4.0        31s             deployed
-```
+### Managing workload clusters
 
 Now the managed application will be installed in your workload cluster. To learn more about how to utilize and configure Managed Apps, please refer to [the documentation]({{< relref "/platform-overview/app-platform/overview">}}).
```

Comment:
I dont understand this paragraph here. Which managed application?

Reply:
This paragraph looks like overflow - managed apps are discussed in the section below, I think this can/should be either deleted or moved to the "Installing managed apps" section.

Reply:
this has been addressed